support-frontend/package.json

```diff
@@ -70,6 +71,37 @@
 		"verbose": true,
 		"testEnvironment": "jest-environment-jsdom-global"
 	},
+  "knip": {
+    "entry": [
```

Comment:
Not necessarily for now, but it'd be great if this was driven off of the same list of entry points we define in `webpack.entryPoints.js` (that way we couldn't forget to add a new entry point in here). It looks like as well as defining in `package.json`, this config can also exist in a separate js config file, from which we _could_ import from `webpack.entryPoints.js`.

Reply:
That's a great idea, it was a pain copying them over! I'll set-up a separate knip config and import the `webpack.entryPoints.js` so we don't miss one if something new gets added!